pilot/pkg/serviceregistry/serviceentry/servicediscovery_test.go

```diff
@@ -408,10 +408,28 @@ func TestServiceDiscoveryServiceUpdate(t *testing.T) {
 			makeInstance(httpStaticOverlayUpdatedNs, "7.7.7.7", 4567, httpStaticOverlayUpdatedNs.Spec.(*networking.ServiceEntry).Ports[0], map[string]string{"namespace": "bar"}, PlainText),
 		}
 		expectServiceInstances(t, sd, httpStaticOverlayUpdatedNs, 0, instances)
-		// svc update is only triggered on deletion. Also expect a full push as the service has changed
+		// 1. svc delete is not triggered on since `httpStatic` is there and has instances, so we should not delete the endpoints shards of "*.google.com".
```

Comment:
"svc delete is not triggered on since" -> 
"svc delete is not triggered since"

Reply:
ok, will remove